lng/en.lng.php

```diff
@@ -1947,6 +1947,14 @@
 			'title' => 'AXFR servers',
 			'description' => 'A comma separated list of IP addresses allowed to transfer (AXFR) dns zones.',
 		],
+		'bindmasterservers' => [
+			'title' => 'Bind9 primary (master) servers',
```

Comment:
I think it could be irritating for users to make a connection from `Bind9 primary (master) servers` to `this actually generates mirror configs`, what do you think? Is there a more understandable/descriptive title/description we could use?